src/app/item-popup/DesktopItemActions.m.scss

```diff
@@ -7,6 +7,15 @@
   background-color: #161626;
 }
 
+.collapsed {
+  border-radius: 7px;
+  margin: 0 5px;
```

Comment:
I think we should keep sharp corners and have the menu "attached", to keep visual consistency with the expanded menu.

Reply:
<img width="480" alt="Screen Shot 2020-10-15 at 11 14 34 AM" src="https://user-images.githubusercontent.com/424158/96170174-1e5bf580-0ed8-11eb-82ab-f946b51ff5a7.png">